geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterDistributionManager.java

```diff
@@ -3225,7 +3061,17 @@ private void scheduleIncomingMessage(DistributionMessage message) {
 
   @Override
   public boolean isAdam() {
```

Comment:
I think that we should change this method to something that will be informative to everyone. And also the javadoc needs to be updated. Maybe it could be something like "wasLonerOnStartup"

Reply:
Yes. I've changed this.